sale_order_general_discount_triple/models/res_config_settings.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    general_discount = fields.Selection(
+        [
+            ("discount", "Discount"),
+            ("discount2", "Discount 2"),
+            ("discount3", "Discount 3"),
+        ],
+        "General Discount",
+        required=True,
+        default="discount",
```

Comment:
Please use config_parameter attribute to remove get/set functions:
```suggestion
        default="discount",
        config_parameter="sale_order_general_discount_triple.general_discount"
```

Reply:
fixed, thanks for pointing out.